hms-api/package.json

```
@@ -4,26 +4,28 @@
   "scripts": {
     "test": "jest",
     "build": "sls package",
-    "start": "sls deploy --stage local",
+    "start": "sls offline start --stage local",
     "stop": "sls remove --stage local"
   },
   "license": "ISC",
   "devDependencies": {
     "@types/jest": "27.5.0",
-    "@types/node": "17.0.38",
-    "@typescript-eslint/eslint-plugin": "5.27.0",
-    "@typescript-eslint/parser": "5.27.0",
+    "@types/node": "17.0.35",
+    "@typescript-eslint/eslint-plugin": "5.26.0",
+    "@typescript-eslint/parser": "5.26.0",
     "eslint": "8.16.0",
     "eslint-config-google": "0.14.0",
     "jest": "27.5.1",
-    "serverless-localstack": "0.4.36",
+    "serverless-offline": "^8.8.0",
```

We can be explicit instead of using ^

```suggestion
    "serverless-offline": "8.8.0",
```